docs/30_commands.md

```diff
@@ -378,6 +378,12 @@ Be aware that the presence of both "image" and "imageUrl" and/or of both "suppor
 miactl marketplace apply { -f file-path }... } [flags]
 ```
 
+### list-versions
+
+List all the available versions of a specific Marketplace item.
+
+The flag `--item-id` or `-i` accepts the `itemId` of the Item.
+
```

Comment:
Are we sure that this is the correct place for the command? It seems in the middle of the `marketplace apply` voice...

Reply:
that's right, fixed it and added the missing Synopsis